exporter/exporter_test.go

```diff
@@ -227,13 +227,14 @@ func TestMongoUpMetric(t *testing.T) {
 	ctx := context.Background()
 
 	type testcase struct {
-		URI  string
-		Want int
+		URI         string
+		clusterRole string
+		Want        int
 	}
 
 	testCases := []testcase{
 		{URI: "mongodb://127.0.0.1:12345/admin", Want: 0},
-		{URI: fmt.Sprintf("mongodb://127.0.0.1:%s/admin", tu.GetenvDefault("TEST_MONGODB_STANDALONE_PORT", "27017")), Want: 1},
+		{URI: fmt.Sprintf("mongodb://127.0.0.1:%s/admin", tu.GetenvDefault("TEST_MONGODB_STANDALONE_PORT", "27017")), Want: 1, clusterRole: "mongod"},
```

Comment:
why don't we test non-standalone instances? I think we need to test all of them, don't we?